src/bundle/Resources/public/js/scripts/admin.search.filters.js

```diff
@@ -154,6 +163,76 @@
             defaultDate
         }));
     };
+    const getUsersList = (value) => {
+        const body = JSON.stringify({
+            ViewInput: {
+                identifier: `find-user-by-name-${value}`,
+                public: false,
+                ContentQuery: {
+                    Criteria: {},
+                    FacetBuilders: {},
+                    SortClauses: {},
+                    Query: {
+                        FullTextCriterion: `${value}*`,
+                        ContentTypeIdentifierCriterion: 'user'
```

Comment:
I have doubts about this `hardcoded` content type identifier. What I was thinking what we can do is:

1. Leave it hardcoded because the user is a special content type and should not be replaced.
2. Do some configuration option to allow developers to define which content types are the `user`.
3. Filter content type that contains `ezuser` field type.

2 and 3 assumes we can send multiple identifiers in `ContentTypeIdentifierCriterion`.

Summon @webhdx and @alongosz for an opinion.

Reply:
Yeah definitely we should'nt hardcode user content type identifier. Configurable option is probably the easiest solution for now.

Reply:
> 2 and 3 assumes we can send multiple identifiers in `ContentTypeIdentifierCriterion`.

Yes, AFAICS comma-separated.

For now configuration option would be better. I don't see any API support for finding ContentTypes with specific FieldDefinitions (Field criterion refers to field name, not type).

Reply:
Yes, this must be configurable per admin ui context.